components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -101,6 +101,12 @@ class BrowserToolbar @JvmOverloads constructor(
             }
         }
 
+    override var displayTitle: Boolean
+        get() = displayToolbar.titleView.isVisible()
+        set(value) {
+            displayToolbar.titleView.visibility = if (value) View.VISIBLE else View.GONE
+        }
```

Comment:
Does this need to be a separate flag or can this be controlled by whether a title is set or not? Not every page has a title so I assume if there's no title then we do not want to display this view?

Reply:
Good point, I assumed every URL had a title.